public/store/LogTypeStore.ts

```diff
@@ -54,11 +54,12 @@ export class LogTypeStore {
         0,
         ruleTypes.length,
         ...logTypes
-          .map(({ category, id, name }) => ({
+          .map(({ category, id, name, source }) => ({
             label: name,
             value: name,
             id,
             category,
+            isStandard: source === 'Standard',
```

Comment:
Nitpick: A constant with an exhaustive list of source options would be helpful. Not blocking.

Reply:
Will address the constants in a follow up PR